articles/partner-twilio-python-how-to-use-voice-sms.md

```diff
@@ -58,7 +61,7 @@ As an example, the following TwiML would convert the text **Hello World** to spe
 
     <?xml version="1.0" encoding="UTF-8" ?>
     <Response>
-       <Say>Hello World</Say>
+      <Say>Hello World</Say>
```

Comment:
Perhaps we could spruce up the prose a bit and use `Hello World!` here and elsewhere instead?